default/stringtables/en.txt

```diff
@@ -11128,6 +11128,8 @@ Energy Compressor
 BLD_SHIPYARD_ENRG_COMP_DESC
 '''This is an upgrade to the [[buildingtype BLD_SHIPYARD_BASE]] and is required for the formation of all energy hulls and all further energy hull shipyard upgrades.
 
+Requiring a high density star to siphon energy from, it can only be built at a system with a [[STAR_BLUE]], [[STAR_WHITE]] or [[STAR_BLACK]] star type.
```

Comment:
"High energy stars" sounds good to me to, so seems to be something we can all agree on. Unless @MatGB wants to ask his friend for better suggestions, @LGM-Doyle, can you put this change in so we can merge?